test/integration/targets/ansible/runme.sh

```diff
@@ -15,3 +15,67 @@ ansible-config list | grep 'DEFAULT_REMOTE_USER'
 
 # 'view' command must fail when config file is missing
 ansible-config -c ./ansible-non-existent.cfg view && exit 1 || echo 'Failure is expected'
+
+################################################################################
+# BEGIN: Test case for ansible-galaxy
+#
+
+# Need a relative custom roles path for testing various scenarios of -p
+galaxy_relative_rolespath="./my/custom/roles/path"
+
+# Use geerlingguy's nginx galaxy role as the test dummy (it was the first
+# one that came to mind, this could be something else if someone prefers)
+galaxy_test_role="geerlingguy.nginx"
+galaxy_test_role_github="https://github.com/geerlingguy/ansible-role-nginx.git"
```

Comment:
In order to avoid network issues, the tarball could be pulled from [ansible-ci-files.s3.amazonaws.com](https://github.com/ansible/community/wiki/Testing%3A-progress-tracker#tests-that-use-external-dependencies-and-therefore-are-occasionally-unstable).

Reply:
@pilou- how is pulling from one location on the internet less error prone than another?

Reply:
AWS is used for some part of the CI. Some tests use the URL previously mentioned. It seems this is the [recommended location](https://github.com/ansible/community/wiki/Testing%3A-progress-tracker#tests-that-use-external-dependencies-and-therefore-are-occasionally-unstable).

Reply:
@pilou- ah, learned something new today. Thanks :)